### PR TITLE
Revert "Bump openjdk from 15.0.2-buster to 17.0.1-buster in /java"

### PR DESCRIPTION
**Pull request by radomirbosak**
Reverts sider/devon_rex#695

This is because we could not determine whether ktlint installs without issues using gradle.

### CODE REVIEW

**Comment by gnakagaw** [comment]
LGTM, but there are some CI errors. Wrong bundler version?

**Comment by gnakagaw** [comment]
A new bundler version(2.2.31) released 20 hours ago. 
https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md

We should  bump version constraint for bundler. We have a automatic process. But, it have not ran yet.
https://github.com/sider/devon_rex/blob/master/.github/workflows/bump-bundler.yml

**Comment by radomirbosak** [comment]
>  We have a automatic process

Nice, I didn't know we have an action for that. I ran it, but it didn't create any PR, so I created one manually.

File: java/Dockerfile [hunk 1]
```diff
@@ -36,7 +36,7 @@ RUN curl -fsSLO --compressed "${GIT_SOURCE_URL}" && \
     make "-j$(nproc)" NO_GETTEXT=1 install
 
 
-FROM openjdk:17.0.1-buster
+FROM openjdk:15.0.2-buster
```
Comment (radomirbosak) [inline java/Dockerfile:39]: > __[Goodcheck]__ Update CHANGELOG.md when you change a based Docker image. ([view](https://sider.review/gh/repos/200181683/pulls/705#issue-6229358))
>
> <details>
>
> | Rule |
> | ---- |
> | `sider.devon_rex.update_changelog_on_based_image_changed` |
>
> You can [close this issue](https://sider.review/gh/repos/200181683/pulls/705#issue-6229358) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

This is a revert commit. The changelog was reflected by removing the line that was there.

**Review by gnakagaw — APPROVED** [review]
:shipit:

**Pull request merged**